docs/learn/smart-contract-developers/create-nft-collection-with-lsp7-tokenId.md

```diff
@@ -0,0 +1,956 @@
+---
+sidebar_label: '🃏 Create NFT Collection with a contract address as tokenId'
+description: 'Learn how to create a NFT collection with a contract address as tokenId'
+sidebar_position: 8
+---
+
+# Create NFT Collection with a contract address as tokenId
+
+:::note
+
+This guide builds on top of a Hardhat project using TypeScript as described in the [Getting Started](./getting-started.md) section.
+
+:::
+
+## Introduction
+
+If you have been looking at our [LSP8 token standard](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md), you probably have noticed that addresses could be used as `tokenIds`. In this guide, we will replicate a real-world example where this could be useful.
+
+## Use Case
+
+Let's take the example of [The Dematerialised](https://thedematerialised.com), and more precisely their KLxENDLESS PHYSITAL COLLECTION. As you can see in the picture below, we have three medaillons of different colors.
+
+![KLxENDLESS PHYSITAL COLLECTION](../../../static/img/learn/klxendless-physital-collection.png)
+
+Each medaillon has a supply limit and each medaillon collection has its own metadata with different attributes. What we will do is create an [LSP8 collection](../../standards/tokens//LSP8-Identifiable-Digital-Asset.md) that will represent the KLxENDLESS PHYSITAL collection and each medaillon a sub-collection (purple, blue or gold) will be represented by a limited supply [LSP7 token](../../standards/tokens//LSP7-Digital-Asset.md).
+
+## Prerequisites
+
+- [Hardhat installed and initialized](./getting-started.md)
+- [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) package installed from npm using `npm i @lukso/lsp-smart-contracts@0.14.0` (or the latest version)
+- [`@erc725/erc725.js`](https://www.npmjs.com/package/@erc725/erc725.js) package installed from npm using `npm i @erc725/erc725js` (we will use it to encode the token metadata)
+
+## Create the LSP7 token
+
+Even though it seems a little bit counter intuitive, we will start by creating the LSP7 token. The reason is that the LSP8 collection will be in charge of deploying the LSP7 sub-collection tokens.
+
+### Specify what needs to be set at deployment
+
+To create the LSP7 token contract, we will need to know what we want to be set at deployment. In our case, we want to set the following:
+
+- the name of the sub-collection
+- the symbol of the sub-collection
+- the address of the owner of the sub-collection (in our case it will the the LSP8 collection contract)
```

Comment:
```suggestion
- the address of the [`owner()`](link/to-contract/abi-docs/for-LSP7#owner-method) that will control the sub-collection contract (in our case this will the LSP8 collection contract)
```